volatility3/cli/volshell/generic.py

```diff
@@ -288,6 +289,21 @@ def display_words(self, offset, count=128, layer_name=None):
         remaining_data = self._read_data(offset, count=count, layer_name=layer_name)
         self._display_data(offset, remaining_data, format_string="H")
 
+    def regex_scan(self, pattern, count=128, layer_name=None):
```

Comment:
Is 128 a reasonable default?  Happy if you think it is, just likely to be harder to change once it's in place, so want to pick a good default.

Reply:
I think so. It matches the others (db, dd, dq etc) and I felt like it would be nice that they were the same. Could make it a variable so that they're all kept in sync?

I played with the idea of rerunning the regex on the result so that it displays the full result of the regex (the regex scanner returns the offset and not the data) but I ended up thinking that it'll just be a little bit messy - a few `.*` in the regex and it could be insane huge.

Reply:
Yep, good plan.